util/fipstools/delocate/delocate.go

```diff
@@ -520,6 +520,12 @@ func (d *delocation) processAarch64Instruction(statement, instruction *node32) (
 					d.redirectors[symbol] = redirector
 					symbol = redirector
 					changed = true
+				} else if didChange && symbolIsLocal && len(offset) > 0 {
+					// didChange happens when the inputFile index is not 0 (first one),
+					// so local symbols in the file need to get changed with suffix(`BCM` + `index`).
+					// The change is to avoid potential collides with other files.
+					// If the offset exists, append the offset.
+					symbol = symbol + offset
```

Comment:
Should `changed` be set to true so that a comment is output before this instruction? (l. 618)

Reply:
`didChange` is already assigned to `changed`

Reply:
Thank you, I hadn't noticed that.